api/lib/infrastructure/monitoring-tools.js

```diff
@@ -1,48 +1,44 @@
 const settings = require('../config');
-const { get, set, update } = require('lodash');
+const { get, set, update, omit } = require('lodash');
 const logger = require('../infrastructure/logger');
 const requestUtils = require('../infrastructure/utils/request-response-utils');
 
 const { AsyncLocalStorage } = require('async_hooks');
 const asyncLocalStorage = new AsyncLocalStorage();
 
-function logInfoWithCorrelationIds(data) {
+function getCorrelationContext() {
   if (settings.hapi.enableRequestMonitoring) {
```

Comment:
```suggestion
  if (!settings.hapi.enableRequestMonitoring) {
    return {};
  }
```
et supprimer le `else`

Reply:
Ca se défend. C'est parti <3